atom/app/atom_content_client.cc

```diff
@@ -75,6 +75,17 @@ std::string AtomContentClient::GetProduct() const {
 void AtomContentClient::AddAdditionalSchemes(
     std::vector<std::string>* standard_schemes,
     std::vector<std::string>* savable_schemes) {
+  auto command_line = base::CommandLine::ForCurrentProcess();
+  auto custom_schemes = command_line->GetSwitchValueASCII(
+      switches::kRegisterStandardSchemes);
+  if (!custom_schemes.empty()) {
+    std::vector<std::string> schemes;
+    base::SplitString(custom_schemes, ',', &schemes);
+    for (const auto& scheme : schemes) {
```

Comment:
We can use a simpler version:

```c++
standard_schemes->insert(standard_schemes.end(), custom_schemes.begin(), custom_schemes.end());
```

I don't think we need to test whether a scheme is empty.